crates/wash-lib/src/start/wasmcloud.rs

```diff
@@ -329,8 +329,8 @@ mod test {
 
     #[tokio::test]
     #[cfg_attr(not(can_reach_github_com), ignore = "github.com is not reachable")]
-    async fn can_download_wasmcloud_burrito() {
-        let download_dir = temp_dir().join("can_download_wasmcloud_burrito");
```

Comment:
Wow, is this a holdover from the elixir days?

Reply:
Yep 😢 Just renaming it because it's strange to see the remnant of the old days